driver/portfolio_runner.py

```diff
@@ -83,11 +83,12 @@ def compute_run_time(timeout, configs, pos):
     print("remaining time: {}".format(remaining_time))
     relative_time = configs[pos][0]
     remaining_relative_time = sum(config[0] for config in configs[pos:])
-    print("config {}: relative time {}, remaining {}".format(
-          pos, relative_time, remaining_relative_time))
     # For the last config we have relative_time == remaining_relative_time, so
     # we use all of the remaining time at the end.
```

Comment:
I'm not sure this comment bears much relation to the following code. Should we remove it?

Reply:
Sure, I think the code is clear enough about what it does.